_posts/2020-12-09-ssl-certificate-revocation-list.adoc

```diff
@@ -0,0 +1,454 @@
+---
+layout: post
+title: 'SSL Certificate Revocation Lists with WildFly'
+date: 2020-12-09
+tags: elytron ssl
+synopsis: An overview of the certificate revocation lists support included in WildFly 22.
+author: szaldana
+---
+
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== An Overview of Certificate Revocation Lists
+
+Certificate Revocation Lists (CRLs) contain a list of certificates that have been revoked
+by the issuing Certificate Authority before their expected expiration date, and therefore should no longer
+be trusted. Elytron previously supported configuring only one Certificate Revocation List.
+
+It is now possible to configure multiple CRL files, which is useful when several certificate authorities are used.
+This blog post assumes previous knowledge on how to configure SSL in Elytron. For more information on how to configure SSL,
+refer to the elytron https://docs.wildfly.org/21/WildFly_Elytron_Security.html#configure-ssltls[documentation.]
```

Comment:
We can update the docs link to 24

Reply:
To ``https://docs.wildfly.org/24/WildFly_Elytron_Security.html#configure-ssltls``?

Reply:
Yes. Thanks!

Reply:
Updated